docs/git-commands.md

```diff
@@ -0,0 +1,63 @@
+# Common Github Commands
```

Comment:
git is an open source software that github leverages for its platform. These are git commands, not github commands.